src/js/constants.js

```diff
@@ -40,5 +40,12 @@ var exports = {
   DNT_POLICY_CHECK_INTERVAL: 1000, // one second
 };
 
+exports.BLOCKED_ACTIONS = {
```

Comment:
Can this become a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)?

(In cases where we see the pattern `{'name': true, ...}` where all the values are `true`, we usually want a set)